### PR TITLE
Scaffold and Testing environment 

### PR DESCRIPTION
**Pull request by dernathul**
# [PT Story](https://www.pivotaltracker.com/story/show/172265972)

## Changes proposed in this pull request:
* Adds basic scaffold for react
* Add Cypress and yarn, also the local host and shorcuts in package.json
## What I have learned working on this feature:
* Focus and pay attention to the course material
* Refreshed knowledge about scaffolding
## Packages/Gems added
-Yarn
-Cypress

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root')
+);
+
+// If you want your app to work offline and load faster, you can change
+// unregister() to register() below. Note this comes with some pitfalls.
+// Learn more about service workers: https://bit.ly/CRA-PWA
```
Comment (ghost) [inline src/index.js]: Please remove all comments

File: README.old.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# bundle_client
+Final Project February 2020
```
Comment (hmvitous) [inline README.old.md]: Delete this file

File: cypress/integration/examples/utilities.spec.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+          resolve('foo')
+        }, 1000)
+      })
+    }
+
+    cy.then(() =>
+    // return a promise to cy.then() that
+    // is awaited until it resolves
+      // @ts-ignore TS7006
+      waitOneSecond().then((str) => {
```
Comment (ghost) [inline cypress/integration/examples/utilities.spec.js]: please remove all unnecessary files

File: cypress.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+    "baseUrl": "http://localhost:3001",
+    "chromeWebSecurity": false
```
Comment (hmvitous) [inline cypress.json]: Delete this line
Its not needed

File: cypress/fixtures/example.json [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (hmvitous) [inline cypress/fixtures/example.json]: delete this file

File: cypress/fixtures/profile.json [hunk 6]
```diff
@@ -0,0 +1,5 @@
+{
+  "id": 8739,
+  "name": "Jane",
+  "email": "jane@example.com"
+}
```
Comment (hmvitous) [inline cypress/fixtures/profile.json]: delete this file

File: cypress/fixtures/users.json [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+    },
+    "phone": "024-648-3804",
+    "website": "ambrose.net",
+    "company": {
+      "name": "Hoeger LLC",
+      "catchPhrase": "Centralized empowering task-force",
+      "bs": "target end-to-end models"
+    }
+  }
+]
```
Comment (hmvitous) [inline cypress/fixtures/users.json]: delete this file

File: cypress/integration/examples/actions.spec.js [hunk 8]
```diff
@@ -0,0 +1,298 @@
...
+    // of the (width, height) of the element
+    cy.get('#scrollable-both').scrollTo('75%', '25%')
+
+    // control the easing of the scroll (default is 'swing')
+    cy.get('#scrollable-vertical').scrollTo('center', { easing: 'linear' })
+
+    // control the duration of the scroll (in ms)
+    cy.get('#scrollable-both').scrollTo('center', { duration: 2000 })
+  })
+})
```
Comment (hmvitous) [inline cypress/integration/examples/actions.spec.js]: delete this file

File: public/index.html [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <title>React App</title>
```
Comment (hmvitous) [inline public/index.html]: Change the title

**Review by hmvitous — CHANGES_REQUESTED** [review]

**Review by ghost — COMMENTED** [review]
Code looks okay

**Review by hmvitous — APPROVED** [review]

**Comment by hmvitous** [comment]
Looks good 
Thanks for changing issues

**Comment by ghost** [comment]
LGTM

**Pull request merged**